spanner/api/SpannerTest/Tests.cs

```diff
@@ -432,6 +432,20 @@ void TestStructParameters()
             Assert.Contains("Imagination", readOutput.Stdout);
         }
 
+        [Fact]
+        void TestDeleteSampleData()
+        {
+            ConsoleOutput output = _spannerCmd.Run("deleteSampleData",
+                _fixture.ProjectId, _fixture.InstanceId, _fixture.DatabaseId);
+            Assert.Contains("Deleted individual rows in Albums.", output.Stdout);
+            Assert.Contains($"2 row(s) deleted from Singers.", output.Stdout);
+            Assert.Contains($"7 row(s) deleted from Singers.", output.Stdout);
+
+            // Put the deleted sample data back.
+            _spannerCmd.Run("insertSampleData",

```

Comment:
What if this fails, it will spoil all other tests possibly. As a setup for this test, can we add data (or better yet, specific tables) for the delete operation to use freely?

Reply:
That's fair enough. I set it up such that the the function that executes "deleteSampleData" sets up the tables and data, then drops the tables afterwards. I did it this way because I just didn't want to introduce more command line functions.

Let me know if this is ok or you'd prefer for me to setup the test data in the test itself. We can also cleanup all this during the refactoring.

Reply:
This is fine for now. On rewrite I'd like to have proper test setup and teardown.